manifests/params.pp

```diff
@@ -20,6 +20,25 @@
   $package_name       = 'vault'
   $package_ensure     = 'installed'
 
+  $checksum_type       = 'sha256'
+  $checksums           = {
```

Comment:
That would be a good start, yes.
Currently I manually version bump but something which could one day be automated.